websites/C/Cracked.io/presence.ts

```diff
@@ -0,0 +1,217 @@
+const presence = new Presence({
+		clientId: "1041324883793154098",
+	}),
+	browingTimestamp = Math.floor(Date.now() / 1000);
+
+async function getStrings() {
+	return presence.getStrings(
+		{
+			browse: "general.browsing",
+			search: "general.searchFor",
+			searchSomething: "general.searchSomething",
+			viewCategory: "general.viewCategory",
+			viewHome: "general.viewHome",
+			buttonViewPage: "general.buttonViewPage",
+			buttonViewProfile: "buttonViewProfile",
+		},
+		await presence.getSetting<string>("lang").catch(() => "en")
+	);
+}
+function capitalizeFirstLetter(string: string) {
+	if (string) {
```

Comment:
Check isn't needed

Reply:
It is cuz it may be pass undefiend values.